tests/smcp_recursive_lock_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"maistra/util"
+	"testing"
+)
+
+func testPilotLock() {
+	if _ , err := util.Shell("oc rsh -c discovery $(oc get pods -l app=pilot --no-headers | awk '{print $1}') curl -v http://localhost:8080/debug/cdsz") ; err != nil {
```

Comment:
@gbaufake are those oc rsh and oc get command supposed to be run in istio-system ns ?  Could you update the commands with -n istio-system ?

Reply:
Sure!